src/main/java/bossmonster/Application.java

```diff
@@ -0,0 +1,19 @@
+package bossmonster;
+
+import bossmonster.IO.Input;
+import bossmonster.IO.Output;
+
+public class Application {
+    private static final Player player = new Player();
+    private static final BossMonster bossMonster = new BossMonster();
+    public static void main(String[] args) {
+        Input.bossHP(bossMonster);
+        Input.playerName(player);
+        Input.playerHPMP(player);
+        Output.startGame();
+    }
+    public static void turn(){
+        Output.battleField(player,bossMonster);
+        Input.playerAttack(player,bossMonster);
+    }
+}
```

Comment:
Application에 작성해주셨네요 👍
혹시 코드 컨벤션이라는 말을 들어보신적 있으신가요?

코드를 작성하는 스타일로 개발자들간에 약속같은 느낌입니다!
개인마다, 팀마다 다를 수 있지만 아래 링크를 첨부할테니 읽어보시고 다른 클래스도 마찬가지로 적용해보시면 좋을거 같아요!

[네이버 핵데이 코드 컨벤션](https://naver.github.io/hackday-conventions-java/)

현재 클래스의 코드를 컨벤션에 맞추면 다음과 같이 될 수 있을거 같아요!

```suggestion
public class Application {

    private static final Player player = new Player();
    private static final BossMonster bossMonster = new BossMonster();
    
    public static void main(String[] args) {
        Input.bossHP(bossMonster);
        Input.playerName(player);
        Input.playerHPMP(player);
        Output.startGame();
    }
    
    public static void turn() {
        Output.battleField(player, bossMonster);
        Input.playerAttack(player, bossMonster);
    }
}
```

Reply:
🫡 코드 컨벤션에 맞춰서 수정해봤습니다!
코드 컨벤션이라는게 생각보다 신경써야하는 부분이 많군요😵‍💫
다음부턴 틀리지 않도록 노력해야겠습니다.😅